nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java

```diff
@@ -137,8 +150,8 @@ public static String getExpectedMailContent(DocumentModel commentDocModel, Docum
     public static String getMailContent(MailMessage mailMessage) {
         String content = mailMessage.getContent();
         if (mailMessage.getContentType().contains("text/html")) {
-            Document document = Jsoup.parse(content);
-            return document.select("body").text();
+            Document document = Jsoup.parse(content.replaceAll("<br/>", "\\\\n"));
+            return document.text().replaceAll("\\s?\\\\n\\s?", "\n").trim();
```

Comment:
We need to have a deeper look at this lines and usage of Jsoup. We may already have such things in Nuxeo.

Can you and/or @RSalem07 have a look about using something existing in Nuxeo or something Nuxeo already depends on it in order to remove the Jsoup dependency + to have something simpler here?

Reply:
I find another way to do the suff without using Jsoup library what if we use `Html2TextConverter` i tried it and it works i get correctly the content the only things is that the converter add `\n` instead of space character. whdy @kevinleturc i i used this converter and if we want we can remove the extra '\n' using a regex